src/test/java/core/basesyntax/strategy/BalanceOperationStrategyTest.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.strategy;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.fruitstorge.FruitStorage;
+import core.basesyntax.fruittransaction.FruitTransaction;
+import core.basesyntax.operations.Operation;
+import org.junit.jupiter.api.Test;
+
+class BalanceOperationStrategyTest {
+
+    @Test
+    void handle() {
```

Comment:
invalid test method name. Please try to follow the common mistakes file and give meaningful names to your test methods